config/app-console.php

```diff
@@ -25,6 +25,7 @@
 			'migrationPath' => [
 				'@migrations',
 				'@yii/rbac/migrations',
+				'@app/extensions/settings/migrations'
 			],
 		],
```

Comment:
settings should be also added to console app, because they are important (replace params)